identity_credential/src/credential/revocation_bitmap_status.rs

```diff
@@ -65,19 +88,57 @@ impl TryFrom<Status> for RevocationBitmapStatus {
 
   fn try_from(status: Status) -> Result<Self> {
     if status.type_ != Self::TYPE {
-      Err(Error::InvalidStatus(format!(
+      return Err(Error::InvalidStatus(format!(
         "expected type '{}', got '{}'",
         Self::TYPE,
         status.type_
-      )))
-    } else if !status.properties.contains_key(Self::INDEX_PROPERTY_NAME) {
-      Err(Error::InvalidStatus(format!(
-        "missing required property '{}'",
-        Self::INDEX_PROPERTY_NAME
-      )))
+      )));
+    }
+
+    let revocation_bitmap_index: &Value =
+      if let Some(revocation_bitmap_index) = status.properties.get(Self::REVOCATION_BITMAP_INDEX_PROPERTY) {
+        revocation_bitmap_index
+      } else {
+        return Err(Error::InvalidStatus(format!(
+          "missing required property '{}'",
+          Self::REVOCATION_BITMAP_INDEX_PROPERTY
+        )));
+      };
+
+    let revocation_bitmap_index: u32 = if let Value::String(index) = revocation_bitmap_index {
+      index.parse::<u32>().map_err(|err| {
+        Error::InvalidStatus(format!(
+          "property '{}' cannot be converted to an unsigned, 32-bit integer: {err}",
```

Comment:
Nit: the error messages for `u32` conversions are inconsistent. See line 72 and 79 above for the index property conversion.

This index property validation code is also duplicated, can it be extracted into a function instead?